boom/bootloader.py

```
@@ -2028,10 +2063,9 @@ def add_opts(opts, append):
             :returns: A string with additional options appended.
             :rtype: string
             """
-            extra = " ".join(append)
-            return f"{opts} {extra}" if append else opts
+            return f"{opts} {" ".join(append)}" if append else opts
```

Pylint (3.3.7)

[error] Parsing failed: 'f-string: expecting '}' (boom.bootloader, line 2066)'

(E0001)